packages/svelte/jest.config.js

```diff
@@ -0,0 +1,7 @@
+const baseConfig = require('../../jest/jest.config.js');
+
+module.exports = {
+  ...baseConfig,
+  testEnvironment: 'jsdom',
+  passWithNoTests: true, // TODO remove once we have something to test and tests
```

Comment:
Just had to add this so that CI/`yarn test` passes as long as we don't have tests